src/app/app.component.html

```diff
@@ -1 +1,6 @@
+<button (click)="openDialog()">Open</button>
+<jsmu-dialog
+	[dialogType]="dialogService.$dialogType | async"
+	*ngIf="dialogService.$isOpen | async"
```

Comment:
The use of services in the template violates the MVC template

Reply:
resolved